tests/dummy/app/templates/application.hbs

```diff
@@ -1,9 +1,35 @@
-{{#materialize-navbar name='ember-cli-materialize'}}
-  <li>{{#link-to 'buttons'}}Buttons{{/link-to}}</li>
-  <li>{{#link-to 'navbar'}}Navbar{{/link-to}}</li>
-  <li>{{#link-to 'cards'}}Cards{{/link-to}}</li>
-  <li>{{#link-to 'collapsible'}}Collapsible{{/link-to}}</li>
-  <li>{{#link-to 'input'}}Input{{/link-to}}</li>
-{{/materialize-navbar}}
+<header>
+  {{#materialize-navbar name='ember-cli-materialize'}}
+    <li>{{#link-to 'buttons'}}Buttons{{/link-to}}</li>
+    <li>{{#link-to 'navbar'}}Navbar{{/link-to}}</li>
+    <li>{{#link-to 'cards'}}Cards{{/link-to}}</li>
+    <li>{{#link-to 'collapsible'}}Collapsible{{/link-to}}</li>
+    <li>{{#link-to 'input'}}Input{{/link-to}}</li>
+  {{/materialize-navbar}}
+</header>
+
+<main>
+  {{outlet}}
+</main>
+
+{{#materialize-footer}}
+  <div class="container">
+    <div class="row">
+      <div class="col l6 s12">
+        <h5 class="white-text">Footer Content</h5>
+        <p class="grey-text text-lighten-4">You can use rows and columns here to organize your footer content.</p>
+      </div>
+      <div class="col l4 offset-l2 s12">
+        <h5 class="white-text">Links</h5>
+        <ul>
+          <li><a class="grey-text text-lighten-3" href="#!">Link 1</a></li>
+          <li><a class="grey-text text-lighten-3" href="#!">Link 2</a></li>
+          <li><a class="grey-text text-lighten-3" href="#!">Link 3</a></li>
+          <li><a class="grey-text text-lighten-3" href="#!">Link 4</a></li>
+        </ul>
+      </div>
+    </div>
+  </div>
```

Comment:
@truenorth, what would you think would be better for this? The markup I'm passing is in just the example markup - I was thinking, as a developer, I wouldn't want to be stuck having my footer be in some predefined format as the example. I also thought that this fit in with how the other components are structured at this time.

This is what you proposed several days ago when I responded I had something similar already done locally:
```
{{#materialize-footer subfooterView="myFooterViewClass"}}
  <div class="col l6 s12">
    <h5 class="white-text">Footer Content</h5>
    <p class="grey-text text-lighten-4">You can use rows and columns here to organize your footer content.</p>
  </div>
  <div class="col l4 offset-l2 s12">
    <h5 class="white-text">Links</h5>
    <ul>
      <li><a class="grey-text text-lighten-3" href="#!">Link 1</a></li>
      <li><a class="grey-text text-lighten-3" href="#!">Link 2</a></li>
      <li><a class="grey-text text-lighten-3" href="#!">Link 3</a></li>
      <li><a class="grey-text text-lighten-3" href="#!">Link 4</a></li>
    </ul>
  </div>
{{/materialize-footer}}
```
Maybe I'm misunderstanding though. When you are saying this is a lot of markup, are you just referencing the container div and the row div (that seems to be the biggest difference between your proposal and this)?
Thanks for the input!